modules/uncertainty/R/ensemble.R

```diff
@@ -222,6 +222,7 @@ write.ensemble.configs <- function(defaults, ensemble.samples, settings, model,
   }
   
   # See if we need to write to DB
+  if (!is.null(settings$database) && !is.null(settings$database$bety) && !is.null(settings$database$bety$write)) {
```

Comment:
If a list entry exists we know all its parents exist too, so I think this can be safely simplified to just 
```suggestion
  if (!is.null(settings$database$bety$write)) {
```